packages/wallets/src/components/WalletTourGuide/WalletTourGuideSettings.tsx

```diff
@@ -13,27 +31,315 @@ export const tourStepConfig: Step[] = [
         ),
         disableBeacon: true,
         disableOverlayClose: true,
-        target: '.wallets-accordion__header:has(+ .wallets-accordion__content--visible)',
+        placement: 'bottom',
+        spotlightPadding: 0,
+        styles: { spotlight: { borderRadius: '0.8rem' } },
+        target: '.wallets-carousel-content__container .wallets-card',
         title: (
             <WalletText color='red' size='sm' weight='bold'>
                 Wallets
             </WalletText>
         ),
     },
+    // Choose Wallets
+    {
```

Comment:
optional: would it make sense to split that into separate files, given chunkiness of this one?

Reply:
I'm not sure, this file has onboarding steps for desktop and responsive and Tooltip component, that's all. I'm not sure what we have to move to another file, maybe just `Tooltip` component? In my opinion it's too small. Separate desktop and mobile steps, maybe, but still not sure it worth it.

Reply:
Sounds good to me - even extracting the mobile-steps and desktop-steps to separate files will be good. I did quick stats on js/ts/jsx/tsx file sizes in project, and while its not the top (stores are top biggest by a far stretch), its still within 3rd-5th percentile of biggest ones (so my gut feeling while reading it was quite correct :D)

Reply:
@wojciech-deriv ok, I'll separate it